static/style.css

```diff
@@ -267,6 +267,96 @@ marquee {
     background-color: rgb(67, 67, 163);
 }
 
+/* Baby and kids section */
+
+.carousel{
+    width:100%;
+    height:500px;
+    /* left: 400px; */
+  
+  }
+  
+   .carousel-item img{
+    height: 500px;
+    width: 300px;
+     
+   }
+   
+  .image p{
+    padding: 20px 50px 20px 50px;
+  }
+  #products{
+    padding: 20px 40px 80px 40px;
+  /*   margin: 0 40px 0 40px; */
+    
+    
+  }
+  
+  .image{
+    padding: 30px 30px 50px 30px;
+    margin-left: 20px;
+    margin-right: 20px;
+    position: relative;
+    display: inline-block;
+    box-shadow: 1rem 1rem 1rem -0.9rem rgb(81, 3, 145);
+    border-radius: 30px;
+  
+    /* margin: auto;
+    text-align: center; */
+  }
+  .image h3{
+    margin-top: 20px;
+    color: rgb(6, 57, 99);
+    
+    
+  }
+  .image p{
+    text-align: center;
+    font-size: 20px;
+    color: purple;
+    padding: 10px 10px 10px 10px;
+    
+  }
+  .price {
```

Comment:
`.babyandkids .price {`

Reply:
I have applied the changes.
Can you check it now ?

![babypage1](https://user-images.githubusercontent.com/56999749/126477405-4e74e9ca-eca2-4624-b977-e9d1baf7fc62.JPG)